.golangci.yaml

```diff
@@ -20,14 +21,11 @@ linters:
   - asciicheck
   - containedctx
   - decorder
-  - depguard
-  - dogsled
   - errcheck
   - errorlint
   - exportloopref
   - goconst
   - gocritic
-  - gocyclo
```

Comment:
Not sure if this is very helpful at all, given that it counts the number of branches in a function/method to determine complexity, which might be a bit arbitrary

Reply:
readded